### ISSUE TITLE
Add Mega All-Contributors File

### ISSUE THREAD

**Issue by madiedgar** [legesher/legesher#65]
## Add All Encompassing All-Contributors File 
We have so many amazing people contributing to our project and want to recognize all of the different types of contribution not just within a single repository but also across the whole organization and project. 

**To complete this issue**:
- [ ] Update [`CONTRIBUTING.md](https://github.com/legesher/legesher/blob/master/community/CONTRIBUTORS.md) to include a larger list of contributors across all of the repositories in this organization.
  - Keep their contributions type the same, and if they have contributed to multiple repositories in different ways include all contribution types. 
  - Documentation may be found [here](https://allcontributors.org/) for all-contributors

Something to think about how we can keep this up to date across all projects 🤔. Also,  how will calls to the all-contributors bot effect both files (`README.md` & the master list in `CONTRIBUTING.md`)?

**Comment by jsonyeung** [comment]
Hi, can I try this issue?

**Comment by madiedgar** [comment]
Yes @jsonyeung! That would be awesome 👏 I'll assign you to it!

**Comment by jsonyeung** [comment]
Thanks @madiedgar ! 😊 Any ideas on how to start this? From what I understand, we want to gather the contributors from all the repos and put them into a single `CONTRIBUTING.md` file?

**Comment by madiedgar** [comment]
Yes that's exactly it @jsonyeung! All of the repositories have a `.all-contributorsrc` file where the contributors config lives and a `README.md` that has the table of the contributors in the **Contributors ✨** section. 


_Here's a couple of helpful guides:_
-  Within the `legesher/community/` folder there are 3 files: `README.md`, `PATRONS.md`, and `CONTRIBUTORS.md`. Add another `.all-contributorsrc` file in this folder.
- You'll want to pull all of the users in the contributors object across all of the other repositories. 
``` md
"contributors": [
    {
      "login": "madiedgar",
      "name": "Madison (Pfaff) Edgar",
      "avatar_url": "https://avatars0.githubusercontent.com/u/7844510?v=4",
      "profile": "http://withmadi.co",
      "contributions": [
        "projectManagement",
        "code"
      ]
    }
]
```
- The most important part of this issue is that the users in the `contributors: []` are all added and the users who have contributed to 2 different repositories have all of the different types of contributions added in their specific`"contributions": []` list.

- In the `README.md`, you'll need to add the comments that will help the all-contributors bot add the proper place in the file.
``` md
<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
<!-- prettier-ignore-start -->
<!-- markdownlint-disable -->
<table>
  <tr>
    <td align="center"><a href="http://withmadi.co"><img src="https://avatars0.githubusercontent.com/u/7844510?v=4" width="100px;" alt="Madison (Pfaff) Edgar"/><br /><sub><b>Madison (Pfaff) Edgar</b></sub></a><br /><a href="https://github.com/madiedgar/legesher/commits?author=madiedgar" title="Code">💻</a> <a href="#projectManagement-madiedgar" title="Project Management">📆</a></td>
    <td align="center"><a href="https://github.com/Beomus"><img src="https://avatars1.githubusercontent.com/u/54792554?v=4" width="100px;" alt="Beomus"/><br /><sub><b>Beomus</b></sub></a><br /><a href="#marketing-Beomus" title="Marketing - People who help in marketing the repo/project">💌</a></td>
    <td align="center"><a href="https://github.com/AlexandruBuhai"><img src="https://avatars3.githubusercontent.com/u/23296075?v=4" width="100px;" alt="Alex Buhai"/><br /><sub><b>Alex Buhai</b></sub></a><br /><a href="#marketing-AlexandruBuhai" title="Marketing - People who help in marketing the repo/project">💌</a></td>
    <td align="center"><a href="https://Korolr.me/"><img src="https://avatars0.githubusercontent.com/u/11715165?v=4" width="100px;" alt="Alexsey Ramzaev"/><br /><sub><b>Alexsey Ramzaev</b></sub></a><br /><a href="https://github.com/madiedgar/legesher/commits?author=korolr" title="Code">💻</a></td>
    <td align="center"><a href="https://github.com/JasperBarzilaij"><img src="https://avatars2.githubusercontent.com/u/12871682?v=4" width="100px;" alt="Jasper Barzilaij"/><br /><sub><b>Jasper Barzilaij</b></sub></a><br /><a href="https://github.com/madiedgar/legesher/commits?author=JasperBarzilaij" title="Documentation">📖</a></td>
  </tr>
</table>

<!-- markdownlint-enable -->
<!-- prettier-ignore-end -->

<!-- ALL-CONTRIBUTORS-LIST:END -->
```

**Comment by madiedgar** [comment]
Closing this as it was resolved by @jsonyeung in #68
